Exception when unmarshalling Devices-related objects in console

Hello all,

By navigating in the console I noticed that several tabs in the Devices section are throwing an exception. I'll attach the stack trace for Packages and Bundles sections.

Packages:
```
15:11:37,919 [http-nio-8080-exec-7] WARN  KapuaExceptionHandler:90  - RPC service non-application error
15:11:37,932 [http-nio-8080-exec-7] WARN  KapuaException:90  - Could not load Exception Messages Bundle for Locale en_US
org.eclipse.kapua.translator.exception.TranslatorException: Error: <?xml version="1.0" encoding="UTF-8"?><packages><package><name>org.eclipse.iot.greenhouse</name><version>0.1.0</version><bundles><bundle><name>org.eclipse.iot.greenhouse.sensors</name><version>0.1.0.SNAPSHOT</version></bundle><bundle><name>org.eclipse.iot.greenhouse.publisher</name><version>0.1.0.SNAPSHOT</version></bundle><bundle><name>org.eclipse.iot.greenhouse.sensors.raspberrypi</name><version>0.1.0.SNAPSHOT</version></bundle><bundle><name>org.eclipse.iot.greenhouse.coap</name><version>0.1.0.SNAPSHOT</version></bundle></bundles></package></packages>
	at org.eclipse.kapua.translator.kura.kapua.TranslatorAppPackageKuraKapua.translate(TranslatorAppPackageKuraKapua.java:173)
	at org.eclipse.kapua.translator.kura.kapua.TranslatorAppPackageKuraKapua.translate(TranslatorAppPackageKuraKapua.java:87)
	at org.eclipse.kapua.translator.kura.kapua.TranslatorAppPackageKuraKapua.translate(TranslatorAppPackageKuraKapua.java:58)
	at org.eclipse.kapua.service.device.management.commons.call.DeviceCallExecutor.send(DeviceCallExecutor.java:104)
	at org.eclipse.kapua.service.device.management.packages.internal.DevicePackageManagementServiceImpl.getInstalled(DevicePackageManagementServiceImpl.java:94)
	at org.eclipse.kapua.app.console.server.GwtDeviceManagementServiceImpl.findDevicePackages(GwtDeviceManagementServiceImpl.java:118)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at com.google.gwt.user.server.rpc.RPC.invokeAndEncodeResponse(RPC.java:569)
	at com.google.gwt.user.server.rpc.RemoteServiceServlet.processCall(RemoteServiceServlet.java:208)
	at com.google.gwt.user.server.rpc.RemoteServiceServlet.processPost(RemoteServiceServlet.java:248)
	at com.google.gwt.user.server.rpc.AbstractRemoteServiceServlet.doPost(AbstractRemoteServiceServlet.java:62)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:648)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:729)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:292)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:207)
	at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:52)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:240)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:207)
	at org.mortbay.servlet.UserAgentFilter.doFilter(UserAgentFilter.java:78)
	at org.mortbay.servlet.GzipFilter.doFilter(GzipFilter.java:131)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:240)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:207)
	at org.apache.shiro.web.servlet.AbstractShiroFilter.executeChain(AbstractShiroFilter.java:449)
	at org.eclipse.kapua.app.console.filter.KapuaWebFilter.executeChain(KapuaWebFilter.java:44)
	at org.apache.shiro.web.servlet.AbstractShiroFilter$1.call(AbstractShiroFilter.java:365)
	at org.apache.shiro.subject.support.SubjectCallable.doCall(SubjectCallable.java:90)
	at org.apache.shiro.subject.support.SubjectCallable.call(SubjectCallable.java:83)
	at org.apache.shiro.subject.support.DelegatingSubject.execute(DelegatingSubject.java:383)
	at org.apache.shiro.web.servlet.AbstractShiroFilter.doFilterInternal(AbstractShiroFilter.java:362)
	at org.apache.shiro.web.servlet.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:125)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:240)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:207)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:212)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:106)
	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:502)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:141)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:79)
	at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:616)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:88)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:528)
	at org.apache.coyote.http11.AbstractHttp11Processor.process(AbstractHttp11Processor.java:1100)
	at org.apache.coyote.AbstractProtocol$AbstractConnectionHandler.process(AbstractProtocol.java:687)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1520)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.run(NioEndpoint.java:1476)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Thread.java:745)
Caused by: java.lang.RuntimeException: javax.xml.bind.JAXBException: 
Descriptor Exceptions: 
---------------------------------------------------------

Exception [EclipseLink-63] (Eclipse Persistence Services - 2.6.3.v20160428-59c81c5): org.eclipse.persistence.exceptions.DescriptorException
Exception Description: The instance creation method [org.eclipse.kapua.model.config.metatype.MetatypeXmlRegistry.newKapuaTscalar], with no parameters, does not exist, or is not accessible.
Internal Exception: java.lang.NoSuchMethodException: org.eclipse.kapua.model.config.metatype.MetatypeXmlRegistry.newKapuaTscalar()
Descriptor: XMLDescriptor(org.eclipse.kapua.model.config.metatype.KapuaTscalar --> [DatabaseTable(ns0:OCD)])

Runtime Exceptions: 
---------------------------------------------------------

 - with linked exception:
[Exception [EclipseLink-0] (Eclipse Persistence Services - 2.6.3.v20160428-59c81c5): org.eclipse.persistence.exceptions.IntegrityException
Descriptor Exceptions: 
---------------------------------------------------------

Exception [EclipseLink-63] (Eclipse Persistence Services - 2.6.3.v20160428-59c81c5): org.eclipse.persistence.exceptions.DescriptorException
Exception Description: The instance creation method [org.eclipse.kapua.model.config.metatype.MetatypeXmlRegistry.newKapuaTscalar], with no parameters, does not exist, or is not accessible.
Internal Exception: java.lang.NoSuchMethodException: org.eclipse.kapua.model.config.metatype.MetatypeXmlRegistry.newKapuaTscalar()
Descriptor: XMLDescriptor(org.eclipse.kapua.model.config.metatype.KapuaTscalar --> [DatabaseTable(ns0:OCD)])

Runtime Exceptions: 
---------------------------------------------------------
]
	at org.eclipse.kapua.app.console.ConsoleJAXBContextProvider.getJAXBContext(ConsoleJAXBContextProvider.java:71)
	at org.eclipse.kapua.commons.util.xml.XmlUtil.get(XmlUtil.java:285)
	at org.eclipse.kapua.commons.util.xml.XmlUtil.unmarshal(XmlUtil.java:197)
	at org.eclipse.kapua.commons.util.xml.XmlUtil.unmarshal(XmlUtil.java:160)
	at org.eclipse.kapua.commons.util.xml.XmlUtil.unmarshal(XmlUtil.java:143)
	at org.eclipse.kapua.translator.kura.kapua.TranslatorAppPackageKuraKapua.translate(TranslatorAppPackageKuraKapua.java:171)
	... 50 more
Caused by: javax.xml.bind.JAXBException: 
Descriptor Exceptions: 
---------------------------------------------------------

Exception [EclipseLink-63] (Eclipse Persistence Services - 2.6.3.v20160428-59c81c5): org.eclipse.persistence.exceptions.DescriptorException
Exception Description: The instance creation method [org.eclipse.kapua.model.config.metatype.MetatypeXmlRegistry.newKapuaTscalar], with no parameters, does not exist, or is not accessible.
Internal Exception: java.lang.NoSuchMethodException: org.eclipse.kapua.model.config.metatype.MetatypeXmlRegistry.newKapuaTscalar()
Descriptor: XMLDescriptor(org.eclipse.kapua.model.config.metatype.KapuaTscalar --> [DatabaseTable(ns0:OCD)])

Runtime Exceptions: 
---------------------------------------------------------

 - with linked exception:
[Exception [EclipseLink-0] (Eclipse Persistence Services - 2.6.3.v20160428-59c81c5): org.eclipse.persistence.exceptions.IntegrityException
Descriptor Exceptions: 
---------------------------------------------------------

Exception [EclipseLink-63] (Eclipse Persistence Services - 2.6.3.v20160428-59c81c5): org.eclipse.persistence.exceptions.DescriptorException
Exception Description: The instance creation method [org.eclipse.kapua.model.config.metatype.MetatypeXmlRegistry.newKapuaTscalar], with no parameters, does not exist, or is not accessible.
Internal Exception: java.lang.NoSuchMethodException: org.eclipse.kapua.model.config.metatype.MetatypeXmlRegistry.newKapuaTscalar()
Descriptor: XMLDescriptor(org.eclipse.kapua.model.config.metatype.KapuaTscalar --> [DatabaseTable(ns0:OCD)])

Runtime Exceptions: 
---------------------------------------------------------
]
	at org.eclipse.persistence.jaxb.JAXBContext$TypeMappingInfoInput.createContextState(JAXBContext.java:1127)
	at org.eclipse.persistence.jaxb.JAXBContext.<init>(JAXBContext.java:192)
	at org.eclipse.persistence.jaxb.JAXBContextFactory.createContext(JAXBContextFactory.java:165)
	at org.eclipse.persistence.jaxb.JAXBContextFactory.createContext(JAXBContextFactory.java:152)
	at org.eclipse.persistence.jaxb.JAXBContextFactory.createContext(JAXBContextFactory.java:112)
	at org.eclipse.persistence.jaxb.JAXBContextFactory.createContext(JAXBContextFactory.java:102)
	at org.eclipse.kapua.app.console.ConsoleJAXBContextProvider.getJAXBContext(ConsoleJAXBContextProvider.java:45)
	... 55 more
Caused by: Exception [EclipseLink-0] (Eclipse Persistence Services - 2.6.3.v20160428-59c81c5): org.eclipse.persistence.exceptions.IntegrityException
Descriptor Exceptions: 
---------------------------------------------------------

Exception [EclipseLink-63] (Eclipse Persistence Services - 2.6.3.v20160428-59c81c5): org.eclipse.persistence.exceptions.DescriptorException
Exception Description: The instance creation method [org.eclipse.kapua.model.config.metatype.MetatypeXmlRegistry.newKapuaTscalar], with no parameters, does not exist, or is not accessible.
Internal Exception: java.lang.NoSuchMethodException: org.eclipse.kapua.model.config.metatype.MetatypeXmlRegistry.newKapuaTscalar()
Descriptor: XMLDescriptor(org.eclipse.kapua.model.config.metatype.KapuaTscalar --> [DatabaseTable(ns0:OCD)])

Runtime Exceptions: 
---------------------------------------------------------

	at org.eclipse.persistence.internal.sessions.DatabaseSessionImpl.initializeDescriptors(DatabaseSessionImpl.java:715)
	at org.eclipse.persistence.internal.sessions.DatabaseSessionImpl.initializeDescriptors(DatabaseSessionImpl.java:651)
	at org.eclipse.persistence.internal.sessions.DatabaseSessionImpl.initializeDescriptors(DatabaseSessionImpl.java:582)
	at org.eclipse.persistence.internal.sessions.DatabaseSessionImpl.postConnectDatasource(DatabaseSessionImpl.java:818)
	at org.eclipse.persistence.internal.sessions.DatabaseSessionImpl.login(DatabaseSessionImpl.java:775)
	at org.eclipse.persistence.oxm.XMLContext$XMLContextState.setupSession(XMLContext.java:749)
	at org.eclipse.persistence.oxm.XMLContext$XMLContextState.setupSession(XMLContext.java:1)
	at org.eclipse.persistence.internal.oxm.Context$ContextState.<init>(Context.java:80)
	at org.eclipse.persistence.oxm.XMLContext$XMLContextState.<init>(XMLContext.java:712)
	at org.eclipse.persistence.oxm.XMLContext.<init>(XMLContext.java:194)
	at org.eclipse.persistence.jaxb.JAXBContext$TypeMappingInfoInput.createContextState(JAXBContext.java:1151)
	at org.eclipse.persistence.jaxb.JAXBContext$TypeMappingInfoInput.createContextState(JAXBContext.java:1124)
	... 61 more
15:11:37,947 [http-nio-8080-exec-7] WARN  KapuaException:230  - Could not load Exception Messages Bundle for Locale en_US
15:11:37,954 [http-nio-8080-exec-7] WARN  KapuaException:230  - Could not load Exception Messages Bundle for Locale en_US
```

Bundles:
```
15:11:48,496 [http-nio-8080-exec-8] WARN  KapuaExceptionHandler:90 kapua-sys - RPC service non-application error
15:11:48,498 [http-nio-8080-exec-8] WARN  KapuaException:90 kapua-sys - Could not load Exception Messages Bundle for Locale en_US
org.eclipse.kapua.translator.exception.TranslatorException: Error: <?xml version="1.0" encoding="UTF-8"?><bundles><bundle><name>org.eclipse.osgi</name><version>3.8.1.v20120830-144521</version><id>0</id><state>ACTIVE</state></bundle><bundle><name>org.eclipse.equinox.cm</name><version>1.0.400.v20120522-1841</version><id>1</id><state>ACTIVE</state></bundle><bundle><name>org.eclipse.equinox.common</name><version>3.6.100.v20120522-1841</version><id>2</id><state>ACTIVE</state></bundle><bundle><name>org.eclipse.equinox.registry</name><version>3.5.200.v20120522-1841</version><id>3</id><state>ACTIVE</state></bundle><bundle><name>org.eclipse.equinox.http.registry</name><version>1.1.200.v20120522-2049</version><id>4</id><state>ACTIVE</state></bundle><bundle><name>org.eclipse.equinox.console</name><version>1.0.0.v20120522-1841</version><id>5</id><state>ACTIVE</state></bundle><bundle><name>org.eclipse.equinox.ds</name><version>1.4.0.v20120522-1841</version><id>6</id><state>ACTIVE</state></bundle><bundle><name>org.eclipse.equinox.event</name><version>1.2.200.v20120522-2049</version><id>7</id><state>ACTIVE</state></bundle><bundle><name>org.eclipse.equinox.io</name><version>1.0.400.v20120522-2049</version><id>8</id><state>ACTIVE</state></bundle><bundle><name>org.eclipse.equinox.metatype</name><version>1.2.0.v20120522-1841</version><id>9</id><state>ACTIVE</state></bundle><bundle><name>org.eclipse.equinox.util</name><version>1.0.400.v20120522-2049</version><id>10</id><state>ACTIVE</state></bundle><bundle><name>org.eclipse.osgi.services</name><version>3.3.100.v20120522-1822</version><id>11</id><state>ACTIVE</state></bundle><bundle><name>org.eclipse.osgi.util</name><version>3.2.300.v20120522-1822</version><id>12</id><state>ACTIVE</state></bundle><bundle><name>org.apache.felix.gogo.command</name><version>0.8.0.v201108120515</version><id>13</id><state>ACTIVE</state></bundle><bundle><name>org.apache.felix.gogo.runtime</name><version>0.8.0.v201108120515</version><id>14</id><state>ACTIVE</state></bundle><bundle><name>org.apache.felix.gogo.shell</name><version>0.8.0.v201110170705</version><id>15</id><state>ACTIVE</state></bundle><bundle><name>org.knowhowlab.osgi.monitoradmin</name><version>1.0.2</version><id>16</id><state>ACTIVE</state></bundle><bundle><name>org.apache.felix.dependencymanager</name><version>3.0.0</version><id>17</id><state>ACTIVE</state></bundle><bundle><name>org.apache.felix.deploymentadmin</name><version>0.9.5</version><id>18</id><state>ACTIVE</state></bundle><bundle><name>osgi.cmpn</name><version>4.3.0.201111022214</version><id>19</id><state>ACTIVE</state></bundle><bundle><name>org.eclipse.equinox.http.jetty</name><version>3.0.0.v20120522-1841</version><id>20</id><state>ACTIVE</state></bundle><bundle><name>org.eclipse.equinox.http.servlet</name><version>1.1.300.v20120522-1841</version><id>21</id><state>ACTIVE</state></bundle><bundle><name>org.eclipse.jetty.continuation</name><version>8.1.3.v20120522</version><id>22</id><state>ACTIVE</state></bundle><bundle><name>org.eclipse.jetty.http</name><version>8.1.3.v20120522</version><id>23</id><state>ACTIVE</state></bundle><bundle><name>org.eclipse.jetty.io</name><version>8.1.3.v20120522</version><id>24</id><state>ACTIVE</state></bundle><bundle><name>org.eclipse.jetty.security</name><version>8.1.3.v20120522</version><id>25</id><state>ACTIVE</state></bundle><bundle><name>org.eclipse.jetty.server</name><version>8.1.3.v20120522</version><id>26</id><state>ACTIVE</state></bundle><bundle><name>org.eclipse.jetty.servlet</name><version>8.1.3.v20120522</version><id>27</id><state>ACTIVE</state></bundle><bundle><name>org.eclipse.jetty.util</name><version>8.1.3.v20120522</version><id>28</id><state>ACTIVE</state></bundle><bundle><name>javax.servlet</name><version>3.0.0.v201112011016</version><id>29</id><state>ACTIVE</state></bundle><bundle><name>org.apache.commons.fileupload</name><version>1.2.2.v20111214-1400</version><id>30</id><state>ACTIVE</state></bundle><bundle><name>org.apache.commons.io</name><version>2.4.0</version><id>31</id><state>ACTIVE</state></bundle><bundle><name>slf4j.api</name><version>1.6.4</version><id>32</id><state>ACTIVE</state></bundle><bundle><name>slf4j.log4j12</name><version>1.6.0</version><id>33</id><state>RESOLVED</state></bundle><bundle><name>log4j</name><version>1.2.17</version><id>34</id><state>RESOLVED</state></bundle><bundle><name>log4j.apache-log4j-extras</name><version>1.1.0</version><id>35</id><state>RESOLVED</state></bundle><bundle><name>org.json</name><version>1.0.0.v201011060100</version><id>36</id><state>ACTIVE</state></bundle><bundle><name>org.hsqldb.hsqldb</name><version>2.3.0</version><id>37</id><state>ACTIVE</state></bundle><bundle><name>com.codeminders.hidapi</name><version>1.1.1</version><id>38</id><state>ACTIVE</state></bundle><bundle><name>org.eclipse.soda.dk.comm</name><version>1.2.1</version><id>39</id><state>ACTIVE</state></bundle><bundle><name>org.apache.commons.lang3</name><version>3.4.0</version><id>40</id><state>ACTIVE</state></bundle><bundle><name>org.usb4java</name><version>1.0.0</version><id>41</id><state>ACTIVE</state></bundle><bundle><name>usb4java-javax</name><version>1.0.0</version><id>42</id><state>ACTIVE</state></bundle><bundle><name>jdk.dio</name><version>1.0.4</version><id>43</id><state>RESOLVED</state></bundle><bundle><name>org.apache.commons.net</name><version>3.1.0.v201205071737</version><id>44</id><state>ACTIVE</state></bundle><bundle><name>org.eclipse.kura.api</name><version>1.0.8</version><id>45</id><state>ACTIVE</state></bundle><bundle><name>org.eclipse.kura.core</name><version>1.0.8</version><id>46</id><state>ACTIVE</state></bundle><bundle><name>org.eclipse.kura.core.certificates</name><version>1.0.2</version><id>47</id><state>ACTIVE</state></bundle><bundle><name>org.eclipse.kura.core.cloud</name><version>1.0.8</version><id>48</id><state>ACTIVE</state></bundle><bundle><name>org.eclipse.kura.core.comm</name><version>1.0.2</version><id>49</id><state>ACTIVE</state></bundle><bundle><name>org.eclipse.kura.core.configuration</name><version>1.0.7</version><id>50</id><state>ACTIVE</state></bundle><bundle><name>org.eclipse.kura.core.crypto</name><version>1.0.3</version><id>51</id><state>ACTIVE</state></bundle><bundle><name>org.eclipse.kura.core.deployment</name><version>1.1.1</version><id>52</id><state>ACTIVE</state></bundle><bundle><name>org.eclipse.kura.core.log</name><version>1.0.1</version><id>53</id><state>ACTIVE</state></bundle><bundle><name>org.eclipse.kura.core.net</name><version>1.0.8</version><id>54</id><state>ACTIVE</state></bundle><bundle><name>org.eclipse.kura.core.status</name><version>1.0.1</version><id>55</id><state>ACTIVE</state></bundle><bundle><name>org.eclipse.kura.deployment.agent</name><version>1.0.6</version><id>56</id><state>ACTIVE</state></bundle><bundle><name>org.eclipse.kura.linux.clock</name><version>1.0.6</version><id>57</id><state>ACTIVE</state></bundle><bundle><name>org.eclipse.kura.linux.command</name><version>1.0.4</version><id>58</id><state>ACTIVE</state></bundle><bundle><name>org.eclipse.kura.linux.position</name><version>1.0.6</version><id>59</id><state>ACTIVE</state></bundle><bundle><name>org.eclipse.kura.linux.usb</name><version>1.0.7</version><id>60</id><state>ACTIVE</state></bundle><bundle><name>org.eclipse.kura.linux.bluetooth</name><version>1.0.4</version><id>61</id><state>ACTIVE</state></bundle><bundle><name>org.eclipse.kura.linux.watchdog</name><version>1.0.5</version><id>62</id><state>ACTIVE</state></bundle><bundle><name>org.apache.camel.camel-core</name><version>2.16.0</version><id>63</id><state>RESOLVED</state></bundle><bundle><name>org.apache.camel.camel-core-osgi</name><version>2.16.0</version><id>64</id><state>RESOLVED</state></bundle><bundle><name>org.apache.camel.camel-kura</name><version>2.16.0</version><id>65</id><state>RESOLVED</state></bundle><bundle><name>org.eclipse.kura.camel</name><version>1.0.0</version><id>66</id><state>ACTIVE</state></bundle><bundle><name>com.gwt.user</name><version>1.0.0</version><id>67</id><state>RESOLVED</state></bundle><bundle><name>org.eclipse.kura.web</name><version>2.0.0</version><id>68</id><state>ACTIVE</state></bundle><bundle><name>org.eclipse.kura.emulator.gpio</name><version>1.0.1</version><id>69</id><state>ACTIVE</state></bundle><bundle><name>org.eclipse.kura.linux.net</name><version>1.0.9</version><id>70</id><state>ACTIVE</state></bundle><bundle><name>org.eclipse.kura.net.admin</name><version>1.0.9</version><id>71</id><state>ACTIVE</state></bundle><bundle><name>org.tigris.mtoolkit.iagent.rpc</name><version>3.0.0.20110411-0918</version><id>72</id><state>ACTIVE</state></bundle><bundle><name>org.eclipse.iot.greenhouse.coap</name><version>0.1.0.SNAPSHOT</version><id>73</id><state>ACTIVE</state></bundle><bundle><name>org.eclipse.iot.greenhouse.publisher</name><version>0.1.0.SNAPSHOT</version><id>74</id><state>ACTIVE</state></bundle><bundle><name>org.eclipse.iot.greenhouse.sensors</name><version>0.1.0.SNAPSHOT</version><id>75</id><state>ACTIVE</state></bundle><bundle><name>org.eclipse.iot.greenhouse.sensors.raspberrypi</name><version>0.1.0.SNAPSHOT</version><id>76</id><state>ACTIVE</state></bundle></bundles>
	at org.eclipse.kapua.translator.kura.kapua.TranslatorAppBundleKuraKapua.translate(TranslatorAppBundleKuraKapua.java:162)
	at org.eclipse.kapua.translator.kura.kapua.TranslatorAppBundleKuraKapua.translate(TranslatorAppBundleKuraKapua.java:83)
	at org.eclipse.kapua.translator.kura.kapua.TranslatorAppBundleKuraKapua.translate(TranslatorAppBundleKuraKapua.java:53)
	at org.eclipse.kapua.service.device.management.commons.call.DeviceCallExecutor.send(DeviceCallExecutor.java:104)
	at org.eclipse.kapua.service.device.management.bundle.internal.DeviceBundleManagementServiceImpl.get(DeviceBundleManagementServiceImpl.java:80)
	at org.eclipse.kapua.app.console.server.GwtDeviceManagementServiceImpl.findBundles(GwtDeviceManagementServiceImpl.java:506)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at com.google.gwt.user.server.rpc.RPC.invokeAndEncodeResponse(RPC.java:569)
	at com.google.gwt.user.server.rpc.RemoteServiceServlet.processCall(RemoteServiceServlet.java:208)
	at com.google.gwt.user.server.rpc.RemoteServiceServlet.processPost(RemoteServiceServlet.java:248)
	at com.google.gwt.user.server.rpc.AbstractRemoteServiceServlet.doPost(AbstractRemoteServiceServlet.java:62)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:648)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:729)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:292)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:207)
	at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:52)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:240)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:207)
	at org.mortbay.servlet.UserAgentFilter.doFilter(UserAgentFilter.java:78)
	at org.mortbay.servlet.GzipFilter.doFilter(GzipFilter.java:131)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:240)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:207)
	at org.apache.shiro.web.servlet.AbstractShiroFilter.executeChain(AbstractShiroFilter.java:449)
	at org.eclipse.kapua.app.console.filter.KapuaWebFilter.executeChain(KapuaWebFilter.java:44)
	at org.apache.shiro.web.servlet.AbstractShiroFilter$1.call(AbstractShiroFilter.java:365)
	at org.apache.shiro.subject.support.SubjectCallable.doCall(SubjectCallable.java:90)
	at org.apache.shiro.subject.support.SubjectCallable.call(SubjectCallable.java:83)
	at org.apache.shiro.subject.support.DelegatingSubject.execute(DelegatingSubject.java:383)
	at org.apache.shiro.web.servlet.AbstractShiroFilter.doFilterInternal(AbstractShiroFilter.java:362)
	at org.apache.shiro.web.servlet.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:125)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:240)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:207)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:212)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:106)
	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:502)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:141)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:79)
	at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:616)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:88)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:528)
	at org.apache.coyote.http11.AbstractHttp11Processor.process(AbstractHttp11Processor.java:1100)
	at org.apache.coyote.AbstractProtocol$AbstractConnectionHandler.process(AbstractProtocol.java:687)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1520)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.run(NioEndpoint.java:1476)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Thread.java:745)
Caused by: java.lang.RuntimeException: javax.xml.bind.JAXBException: 
Descriptor Exceptions: 
---------------------------------------------------------

Exception [EclipseLink-63] (Eclipse Persistence Services - 2.6.3.v20160428-59c81c5): org.eclipse.persistence.exceptions.DescriptorException
Exception Description: The instance creation method [org.eclipse.kapua.model.config.metatype.MetatypeXmlRegistry.newKapuaTscalar], with no parameters, does not exist, or is not accessible.
Internal Exception: java.lang.NoSuchMethodException: org.eclipse.kapua.model.config.metatype.MetatypeXmlRegistry.newKapuaTscalar()
Descriptor: XMLDescriptor(org.eclipse.kapua.model.config.metatype.KapuaTscalar --> [DatabaseTable(ns0:OCD)])

Runtime Exceptions: 
---------------------------------------------------------

 - with linked exception:
[Exception [EclipseLink-0] (Eclipse Persistence Services - 2.6.3.v20160428-59c81c5): org.eclipse.persistence.exceptions.IntegrityException
Descriptor Exceptions: 
---------------------------------------------------------

Exception [EclipseLink-63] (Eclipse Persistence Services - 2.6.3.v20160428-59c81c5): org.eclipse.persistence.exceptions.DescriptorException
Exception Description: The instance creation method [org.eclipse.kapua.model.config.metatype.MetatypeXmlRegistry.newKapuaTscalar], with no parameters, does not exist, or is not accessible.
Internal Exception: java.lang.NoSuchMethodException: org.eclipse.kapua.model.config.metatype.MetatypeXmlRegistry.newKapuaTscalar()
Descriptor: XMLDescriptor(org.eclipse.kapua.model.config.metatype.KapuaTscalar --> [DatabaseTable(ns0:OCD)])

Runtime Exceptions: 
---------------------------------------------------------
]
	at org.eclipse.kapua.app.console.ConsoleJAXBContextProvider.getJAXBContext(ConsoleJAXBContextProvider.java:71)
	at org.eclipse.kapua.commons.util.xml.XmlUtil.get(XmlUtil.java:285)
	at org.eclipse.kapua.commons.util.xml.XmlUtil.unmarshal(XmlUtil.java:197)
	at org.eclipse.kapua.commons.util.xml.XmlUtil.unmarshal(XmlUtil.java:160)
	at org.eclipse.kapua.commons.util.xml.XmlUtil.unmarshal(XmlUtil.java:143)
	at org.eclipse.kapua.translator.kura.kapua.TranslatorAppBundleKuraKapua.translate(TranslatorAppBundleKuraKapua.java:159)
	... 50 more
Caused by: javax.xml.bind.JAXBException: 
Descriptor Exceptions: 
---------------------------------------------------------

Exception [EclipseLink-63] (Eclipse Persistence Services - 2.6.3.v20160428-59c81c5): org.eclipse.persistence.exceptions.DescriptorException
Exception Description: The instance creation method [org.eclipse.kapua.model.config.metatype.MetatypeXmlRegistry.newKapuaTscalar], with no parameters, does not exist, or is not accessible.
Internal Exception: java.lang.NoSuchMethodException: org.eclipse.kapua.model.config.metatype.MetatypeXmlRegistry.newKapuaTscalar()
Descriptor: XMLDescriptor(org.eclipse.kapua.model.config.metatype.KapuaTscalar --> [DatabaseTable(ns0:OCD)])

Runtime Exceptions: 
---------------------------------------------------------

 - with linked exception:
[Exception [EclipseLink-0] (Eclipse Persistence Services - 2.6.3.v20160428-59c81c5): org.eclipse.persistence.exceptions.IntegrityException
Descriptor Exceptions: 
---------------------------------------------------------

Exception [EclipseLink-63] (Eclipse Persistence Services - 2.6.3.v20160428-59c81c5): org.eclipse.persistence.exceptions.DescriptorException
Exception Description: The instance creation method [org.eclipse.kapua.model.config.metatype.MetatypeXmlRegistry.newKapuaTscalar], with no parameters, does not exist, or is not accessible.
Internal Exception: java.lang.NoSuchMethodException: org.eclipse.kapua.model.config.metatype.MetatypeXmlRegistry.newKapuaTscalar()
Descriptor: XMLDescriptor(org.eclipse.kapua.model.config.metatype.KapuaTscalar --> [DatabaseTable(ns0:OCD)])

Runtime Exceptions: 
---------------------------------------------------------
]
	at org.eclipse.persistence.jaxb.JAXBContext$TypeMappingInfoInput.createContextState(JAXBContext.java:1127)
	at org.eclipse.persistence.jaxb.JAXBContext.<init>(JAXBContext.java:192)
	at org.eclipse.persistence.jaxb.JAXBContextFactory.createContext(JAXBContextFactory.java:165)
	at org.eclipse.persistence.jaxb.JAXBContextFactory.createContext(JAXBContextFactory.java:152)
	at org.eclipse.persistence.jaxb.JAXBContextFactory.createContext(JAXBContextFactory.java:112)
	at org.eclipse.persistence.jaxb.JAXBContextFactory.createContext(JAXBContextFactory.java:102)
	at org.eclipse.kapua.app.console.ConsoleJAXBContextProvider.getJAXBContext(ConsoleJAXBContextProvider.java:45)
	... 55 more
Caused by: Exception [EclipseLink-0] (Eclipse Persistence Services - 2.6.3.v20160428-59c81c5): org.eclipse.persistence.exceptions.IntegrityException
Descriptor Exceptions: 
---------------------------------------------------------

Exception [EclipseLink-63] (Eclipse Persistence Services - 2.6.3.v20160428-59c81c5): org.eclipse.persistence.exceptions.DescriptorException
Exception Description: The instance creation method [org.eclipse.kapua.model.config.metatype.MetatypeXmlRegistry.newKapuaTscalar], with no parameters, does not exist, or is not accessible.
Internal Exception: java.lang.NoSuchMethodException: org.eclipse.kapua.model.config.metatype.MetatypeXmlRegistry.newKapuaTscalar()
Descriptor: XMLDescriptor(org.eclipse.kapua.model.config.metatype.KapuaTscalar --> [DatabaseTable(ns0:OCD)])

Runtime Exceptions: 
---------------------------------------------------------

	at org.eclipse.persistence.internal.sessions.DatabaseSessionImpl.initializeDescriptors(DatabaseSessionImpl.java:715)
	at org.eclipse.persistence.internal.sessions.DatabaseSessionImpl.initializeDescriptors(DatabaseSessionImpl.java:651)
	at org.eclipse.persistence.internal.sessions.DatabaseSessionImpl.initializeDescriptors(DatabaseSessionImpl.java:582)
	at org.eclipse.persistence.internal.sessions.DatabaseSessionImpl.postConnectDatasource(DatabaseSessionImpl.java:818)
	at org.eclipse.persistence.internal.sessions.DatabaseSessionImpl.login(DatabaseSessionImpl.java:775)
	at org.eclipse.persistence.oxm.XMLContext$XMLContextState.setupSession(XMLContext.java:749)
	at org.eclipse.persistence.oxm.XMLContext$XMLContextState.setupSession(XMLContext.java:1)
	at org.eclipse.persistence.internal.oxm.Context$ContextState.<init>(Context.java:80)
	at org.eclipse.persistence.oxm.XMLContext$XMLContextState.<init>(XMLContext.java:712)
	at org.eclipse.persistence.oxm.XMLContext.<init>(XMLContext.java:194)
	at org.eclipse.persistence.jaxb.JAXBContext$TypeMappingInfoInput.createContextState(JAXBContext.java:1151)
	at org.eclipse.persistence.jaxb.JAXBContext$TypeMappingInfoInput.createContextState(JAXBContext.java:1124)
	... 61 more
15:11:48,512 [http-nio-8080-exec-8] WARN  KapuaException:230 kapua-sys - Could not load Exception Messages Bundle for Locale en_US
15:11:48,514 [http-nio-8080-exec-8] WARN  KapuaException:230 kapua-sys - Could not load Exception Messages Bundle for Locale en_US
```

--- a/commons/src/main/java/org/eclipse/kapua/commons/configuration/metatype/TscalarImpl.java
+++ b/commons/src/main/java/org/eclipse/kapua/commons/configuration/metatype/TscalarImpl.java
@@ -20,8 +20,11 @@
 
 package org.eclipse.kapua.commons.configuration.metatype;
 
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
 import javax.xml.bind.annotation.XmlEnum;
 import javax.xml.bind.annotation.XmlEnumValue;
+import javax.xml.bind.annotation.XmlRootElement;
 import javax.xml.bind.annotation.XmlType;
 
 import org.eclipse.kapua.model.config.metatype.KapuaTscalar;
@@ -56,6 +59,8 @@ import org.eclipse.kapua.model.config.metatype.KapuaTscalar;
  */
 @XmlEnum
 @XmlType(name = "Tscalar")
+@XmlRootElement(name = "OCD", namespace = "http://www.osgi.org/xmlns/metatype/v1.2.0")
+@XmlAccessorType(XmlAccessType.PROPERTY)
 public enum TscalarImpl implements KapuaTscalar {
     @XmlEnumValue("String")
     STRING("String"),

--- a/console/src/main/java/org/eclipse/kapua/app/console/ConsoleJAXBContextProvider.java
+++ b/console/src/main/java/org/eclipse/kapua/app/console/ConsoleJAXBContextProvider.java
@@ -62,7 +62,6 @@ public class ConsoleJAXBContextProvider implements JAXBContextProvider {
                         KapuaTicon.class,
                         KapuaToption.class,
                         TscalarImpl.class,
-                        KapuaTscalar.class,
                         MetatypeXmlRegistry.class
                 }, null);
             }

--- a/service/api/src/main/java/org/eclipse/kapua/model/config/metatype/KapuaTscalar.java
+++ b/service/api/src/main/java/org/eclipse/kapua/model/config/metatype/KapuaTscalar.java
@@ -51,9 +51,6 @@ import javax.xml.bind.annotation.XmlType;
  * @since 1.0
  * 
  */
-@XmlRootElement(name = "OCD", namespace = "http://www.osgi.org/xmlns/metatype/v1.2.0")
-@XmlAccessorType(XmlAccessType.PROPERTY)
-@XmlType(factoryClass = MetatypeXmlRegistry.class, factoryMethod = "newKapuaTscalar")
 public interface KapuaTscalar {
 
     /**

